cves/CVE-2011-3944.yml

```diff
@@ -253,8 +253,8 @@ interesting_commits:
       * A change that looks really sketchy
       * Commits where a developer mentions security in their message or comments
   commits:
-  - commit:
-    note:
+  - commit: 348efc18364a9daca569ed77e184b843d9bc5f11
+    note: The entire smacker file was created here as well as changes in the systems. This is not the most secure way to add changes to the GitHub/SVN. In additon, this was commited back in 2006, it was in the code 6 years before a Google employee found the exploit
```

Comment:
You should say "The entire smacker.c file". Smacker files are types of video files (that are being parsed in smacker.c). 
saying smacker file could confuse the reader.

Reply:
I will change it to The entire smacker.c file